docs/changes.rst

```diff
@@ -17,6 +17,13 @@ Changelog
 
 - (add entry here)
 
+`3.4` - unreleased
```

Comment:
I would leave this change to the commit which bumps the version and issues a release

Reply:
We have a template for this a few lines above, i.e the original intention was to fill this template (IRC)

Reply:
Updated 👍

Reply:
Thanks!

Reply:
Oh, I meant that the changelog entry should still be in this PR, but placed above what it used to be in the original commit in the templated block

Reply:
Ahh my bad ! Just updated it again